core/modules/catalog/components/ProductBundleOptions.ts

```diff
@@ -3,19 +3,6 @@ import * as types from '../store/product/mutation-types'
 import rootStore from '@vue-storefront/store'
 import i18n from '@vue-storefront/i18n'
 
-function _defaultOptionValue (co, field = 'id') {
-  if (co.product_links && co.product_links.length) {
-    const defaultOption = co.product_links.find(pl => { return pl.is_default })
-    if (defaultOption) {
```

Comment:
Don't we need this logic - I mean the case of field === '*'? I don't see it anywhere in the newly proposed solution